src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -66,6 +68,11 @@
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
+
+    private static final ExecutorService pool = Executors.newFixedThreadPool(
+        Integer.parseInt(System.getProperty("GhprbTrigger.poolSize", "5"))
```

Comment:
is this a config that can be set in the UI or does this have to be passed in as a java arg?